pkg/ui/workspaces/db-console/src/redux/indexUsageStats/indexUsageStatsSagas.ts

```diff
@@ -36,7 +36,7 @@ export const selectIndexStatsKeys = createSelector(
 );
 
 export const KeyToTableRequest = (key: string): TableIndexStatsRequest => {
-  const s = key.split("/");
+  const s = key?.split("/");
```

Comment:
IDK if this is the right way to approach addressing these issues. How come we can't track down where we actually might be reading `null` or `undefined` in the issues linked? Now we have issues where this will case `s` to be null, and we're accessing `s[0]` right below this.

I think there's a lint rule we can introduce that will warn us if we're reading a potentially null field. I'd prefer tracking down the cause in the issues linked, addressing just those, and then introducing the lint rule and fixing the problems that is flagged by that. WDYT?

Reply:
The error message itself doesn't have enough info that allow us to know what is the exact place of the error, so it would be a guess at this point, so this is why I added on some other places too. 
I can do another pass over all the code to make sure I didn't added on places that could cause an issue like the one you pointed.
We can add a lint for that, but the end result would still be adding the checks on all places. I will keep this PR focused on the check itself and look into the lint in another PR.

Reply:
Sounds good! I'd just like to avoid a world where we always use `?` even unnecessarily, instead of properly typing things as nullish / not nullish and respecting that. As long as something like this does not become the norm and it's just a bandaid for the existing issues.